source/leuven/wice_advanced_guide.rst

```diff
@@ -9,15 +9,68 @@ wICE advanced guide
-recommended to compile a version of your code specifically for wICE. A good approach
-to compile on any cluster (such as wICE) is to launch an interactive job (with the
-``srun`` command).
-
-Many dependencies you might need are centrally installed. The modules that are
-optimized for wICE are available when the appropriate
+Compared to the SkyLake and CascadeLake CPUs on Genius, the wICE nodes
+feature more recent CPU models such as Intel IceLake, Intel Sapphire Rapids
+and AMD Genoa. While architectural differences between SkyLake and CascadeLake
+CPUs can be neglected, the differences with newer CPU models are more
+substantial. When it comes to GPUs there are also significant differences in
+the capabilities of P100, V100, A100 and H100 GPUs.
+
+When locally installing software yourself, we therefore recommend to have
+separate installations for the different types of CPUs and (if applicable)
+GPUs on which you intend to run the software. This rule applies most strongly
+to performance-critical code which is compiled from source (and less strongly
+to light-weight tools based on precompiled binaries).
+
+.. note::
+
+    Remember that precompiled binaries (as is often the case when e.g. Conda
+    or PyPI are involved) are not guaranteed to deliver optimal performance
+    for the target device. In case of doubt, performance-critical parts of
+    an application should not rely on precompiled binaries and instead use
+    optimized binaries as provided by the centrally installed modules and/or
+    by local installations from source. See also :ref:`conda for Python` and
+    :ref:`wice_conda`.
+
+To let jobs use the correct installation at runtime, you can make use of
+predefined environment variables such as ``${VSC_ARCH_LOCAL}`` (and possibly
+``${VSC_ARCH_SUFFIX}`` and ``${VSC_INSTITUTE_CLUSTER}``) to organize your
+installations.
+
+For software using CPUs, the different installations would be:
+
+- one for SkyLake and CascadeLake CPUs
+  :raw-html:`<br />`
+  (``${VSC_ARCH_LOCAL}`` = ``skylake`` or ``cascadelake``)
+- one for IceLake CPUs
+  :raw-html:`<br />`
+  (``${VSC_ARCH_LOCAL}`` = ``icelake``)
+- one for Sapphire Rapids CPUs
+  :raw-html:`<br />`
+  (``${VSC_ARCH_LOCAL}`` = ``sapphirerapids``)
+
+For software which also uses GPUs, this would be:
+
+- one for SkyLake CPUs with P100 GPUs
+  :raw-html:`<br />`
+  (``${VSC_ARCH_LOCAL}`` = ``skylake``)
+- one for CascadeLake CPUs with V100 GPUs
+  :raw-html:`<br />`
+  (``${VSC_ARCH_LOCAL}`` = ``cascadelake``)
+- one for IceLake CPUs with A100 GPUs
+  :raw-html:`<br />`
+  (``${VSC_ARCH_LOCAL}`` = ``icelake``)
+- one for AMD Genoa CPUs with H100 GPUs
+  :raw-html:`<br />`
+  (``${VSC_ARCH_LOCAL}`` = ``zen4`` and ``${VSC_ARCH_SUFFIX}`` = ``-h100``)
+
+Compiling software for a particular CPU/GPU model can be done in an
+interactive job (``srun ...``) submitted to a partition containing
+the target device(s) (see e.g. :ref:`genius hardware` and
+:ref:`wice hardware`).
```

Comment:
target CPU's/GPU's instead of devices?

Reply:
(changed as part of 5bfa41d2c2a706327ab763c179759536d3c6d765)